src/rust/lib/graph-editor/src/lib.rs

```diff
@@ -1501,6 +1501,12 @@ fn new_graph_editor(world:&World) -> GraphEditor {
     let outputs        = UnsealedFrpOutputs::new();
     let sealed_outputs = outputs.seal(); // Done here to keep right eval order.
 
+    // === Mouse Cursor Transform ===
+    frp::extend! { network
+        mouse_pos_in_scene <- cursor.frp.position.map(f!((position) {
```

Comment:
we have two names. Mosue and cursor. In my previous PR I was telling that you are using wrong name to implement the logic. Now you are using name "mouse" to refer to "cursor". Please, be very careful regarding names!